website/content/docs/ecs/architecture.mdx

```diff
@@ -39,6 +39,142 @@ This diagram shows the timeline of a task starting up and all its containers:
 - **T1:** The `sidecar-proxy` container starts. It runs Envoy by executing `envoy -c <path-to-bootstrap-json>`.
 - **T2:** The `sidecar-proxy` container is marked as healthy by ECS. It uses a health check that detects if its public listener port is open. At this time, your application containers are started since all Consul machinery is ready to service requests. The only running containers are `consul-client`, `sidecar-proxy`, and your application container(s).
 
+### Task Shutdown
+
+Graceful shutdown is supported when deploying Consul on ECS, which means the following:
+
+* No incoming traffic from the mesh is directed to this task during shutdown.
+* Outgoing traffic to the mesh is possible during shutdown.
+
+This diagram shows an example timeline of a task shutting down:
+
+<img alt="Task Shutdown Timeline" src="/img/ecs-task-shutdown.svg" style={{display: "block", maxWidth: "400px"}} />
+
+- **T0**: ECS sends a TERM signal to all containers.
+  - `consul-client` begins to gracefully leave the Consul cluster, since it is configured with `leave_on_terminate = true`
+  - `health-sync` stops syncing health status from ECS into Consul checks.
+  - `sidecar-proxy` ignores the TERM signal and continues running until it notices that `user-app` container has exited. This allows the application container to continue to make outgoing requests through the proxy to the mesh. This possible due to an entrypoint override for the container, `consul-ecs envoy-entrypoint`.
```

Comment:
```suggestion
  - `sidecar-proxy` ignores the TERM signal and continues running until the `user-app` container exits. This allows the application container to continue making outgoing requests through the proxy to the mesh. `sidecar-proxy` is able to make the outgoing requests because the `consul-ecs envoy-entrypoint` property is configured for the container. The property overrides the entry point. 
```
I'm not entirely clear on what's happening. This is my interpretation. Is the `consul-ecs envoy-entrypoint` configured by default in such a way that enables this behavior? Is this also configuration information that should be mentioned before describing the process.